mh/pat_item.py

```diff
@@ -773,12 +773,20 @@ class LayerBinaryInfo(PatData):
     )
 
 
+def patdata_extender(unpacker):
+    """PatData classes must be defined above this function."""
+    for name, value in globals().items():
+        if not isinstance(value, type):
```

Comment:
I'm sorry if it's obvious but from where `type` come from?

Reply:
It's a kinda obscure concept in python. `type` is a class. This class is used for debugging and meta-programming. `type` returns the type of an object and can also be used to create a class on-the-fly programmatically.

`type` used to check an object's type:
![image](https://user-images.githubusercontent.com/7890055/161385511-5360ca4f-47c5-43ec-b63e-60e2a9a70a79.png)


Here I need to check that the value is a valid python class (otherwise `issubclass` will throw an exception):
 - https://stackoverflow.com/a/395741

Reply:
Ahhhhh, I see